libs/common/src/sql-db-connection/migrations/1732729037950-AddFeeRecordCorrectionTransientFormDataTable.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeeRecordCorrectionTransientFormDataTable1732729037950 implements MigrationInterface {
+  name = 'AddFeeRecordCorrectionTransientFormDataTable1732729037950';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TABLE "FeeRecordCorrectionTransientFormData" (
+                "lastUpdatedAt" datetime2 NOT NULL CONSTRAINT "DF_0de75775c9857115acce6fce573" DEFAULT getdate(),
+                "lastUpdatedByPortalUserId" nvarchar(255),
+                "lastUpdatedByTfmUserId" nvarchar(255),
+                "lastUpdatedByIsSystemUser" bit NOT NULL CONSTRAINT "DF_48cf4c538aa060aacd4c85ce00c" DEFAULT 0,
```

Comment:
One thing we have done before when creating tables is moving these columns manually to the be the last columns, it makes it slightly nicer when querying the database because `SELECT * FROM Table` seems to default to returning the columns in this order and this information is not often the most important when querying